src/components/analytics/AnalyticsCharts.jsx

```diff
@@ -12,32 +11,53 @@ import AnalyticsApiService from './data/service';
 // eslint-disable-next-line no-unused-vars
 export default function AnalyticsCharts(enterpriseId) {
   const [token, setToken] = useState(null);
+  const [tokenUsedOnce, setTokenUsedOnce] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   const [error, setError] = useState(null);
-  // Fetch token
+  const ref = useRef(null);
+  let tableauUrl = 'https://enterprise-tableau.edx.org/views/enterpriseadminanalytics/enroll_dash';
+  if (configuration.TABLEAU_URL) {
```

Comment:
I think we can remove this `if`? I believe `TABLEAU_URL` should always be there.

Reply:
lets just keep it for safety